kvdb/etcd/stm.go

```diff
@@ -452,20 +652,8 @@ func (s *stm) Get(key string) ([]byte, error) {
 		return []byte(put.val), nil
 	}
 
-	// Populate read set if key is present in
```

Comment:
So a main difference here is that the prefetch set is declared up front rather than reduced from the initial read and write sets?

Reply:
Yeah, the original code is a fork of etcd's own STM implementation which has a separate prefetch set to optimize for when we read less on next retry (to reduce the number of cmp ops on commit). 

This (that we read less upon retry) isn't very common in our case so I though it might be a good idea to reduce complexity and just get rid of the prefetch. From my tests it seems that we actually retry very-very rarely, so imho it's not a super important optimization to have.

Reply:
deduced*